android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/AddSwitchChainNetworkFragment.java

```diff
@@ -100,22 +100,11 @@ public View onCreateView(
         Button btnAddSwitchNetwork = view.findViewById(R.id.fragment_add_switch_eth_chain_btn_sign);
         if (mPanelType == ADD_ETHEREUM_CHAIN) {
             btnAddSwitchNetwork.setText(R.string.approve);
-            SpannableString addChainHeading = Utils.createSpannableString(
-                    getString(R.string.brave_wallet_allow_add_network_description),
-                    getString(R.string.brave_wallet), new ClickableSpan() {
-                        @Override
-                        public void onClick(@NonNull View widget) {
-                            TabUtils.openUrlInNewTab(false, Utils.BRAVE_SUPPORT_URL);
-                            TabUtils.bringChromeTabbedActivityToTheTop(getActivity());
-                        }
-                    }, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
-
```

Comment:
This is suppose to be there, just the link text should be "Learn more" instead of brave wallet
https://www.figma.com/file/jZrVG7dsi5npXFTDC6LDGn/Crypto?node-id=1171%3A49854
and should also cross check with desktop.

Reply:
done